1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,24 @@
 function readingList(books) {
-  // Write your code here...
+  for (i = 0; i <= books.length; i++) {
+    let liElement = document.createElement("li");
+    let pElement = document.createElement("p");
+    let bookInfo = books[i].title + " by " + books[i].author;
```

Comment:
Nice one! just a suggestion for future practice: you can use template literals instead of concatenating strings. Template literals make it easier to read and write complex strings.